src/components/Switch/Switch.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from 'test-utils';
+import Switch from './Switch';
+
+describe('Switch', () => {
```

Comment:
minor: while running the test I saw a warning around `getComputedStyle`. Can we include this in `jest/config.js` in order to make the warnings go away?

```js
// jest-dom does not properly implement this method and this workaround
// keeps the test runner from throwing warnings
const { getComputedStyle } = window;
window.getComputedStyle = elt => getComputedStyle(elt);
```